python/python-core/src/main/python/_solver.py

```diff
@@ -34,19 +34,22 @@ class BestSolutionChangedEvent(Generic[Solution_]):
         - In real-time planning, not all ProblemChanges might be processed:
           check `is_every_problem_change_processed`.
 
-        - This `planning_solution` might be uninitialized: check `Score.init_score`.
+        - This `planning_solution` might be uninitialized: check `is_new_best_solution_initialized`.
 
         - This `planning_solution` might be infeasible: check `Score.is_feasible`.
 
+    is_new_best_solution_initialized : bool
```

Comment:
Please update the related test case to verify the new flag.

Reply:
There is no test coverage for this event anywhere in the Python solver.
I think I know why - these tests are notoriously unreliable (flaky), even in the Java solver.
I will not be adding one this time.